csharp-mode.el

```diff
@@ -2508,7 +2508,17 @@ are the string substitutions (see `format')."
 	     t)
 	   (c-put-font-lock-face start (1+ start) 'font-lock-warning-face)))))
 
-(defun c-looking-at-inexpr-block (lim containing-sexp &optional check-at-end)
+(advice-add 'c-looking-at-inexpr-block
+	    :around 'csharp--c-looking-at-inexpr-block-hack)
+
+(defun csharp--c-looking-at-inexpr-block-hack (orig-fun &rest args)
+  (apply
+   (if (and (boundp 'csharp-mode) csharp-mode)
+       csharp--c-looking-at-inexpr-block
```

Comment:
How can csharp-mode not be bound ... If this advice is loaded, and thus csharp-mode too?